package.json

```diff
@@ -32,6 +32,7 @@
     ]
   },
   "dependencies": {
+    "@cosmicdapp/logic": "^0.4.5",
```

Comment:
This is unused, right? I think you can just remove the "Adds new dependencies" commit from this PR